google/cloud/storage/bucket_metadata.h

```diff
@@ -212,18 +212,25 @@ struct BucketIamConfiguration {
   ///     `uniform_bucket_level_access`
   absl::optional<BucketPolicyOnly> bucket_policy_only;
   absl::optional<UniformBucketLevelAccess> uniform_bucket_level_access;
+  absl::optional<std::string> public_access_prevention;
```

Comment:
Is it typical to use a plain string for a field like this in C++? For some languages, we've used an Enum or at least supplied constants for the valid values.

Reply:
> Is it typical to use a plain string for a field like this in C++?

Yes, we do that for all the other fields.

> For some languages, we've used an Enum or at least supplied constants for the valid values.

I can add the constants (strings), you are going to have a bad day with enums when they add a new value.

Reply:
Constants added.